logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/DeleteMessageUseCase.kt

```diff
@@ -55,24 +55,7 @@ class DeleteMessageUseCase(
             )
             messageSender.sendMessage(message)
         }
-            .onSuccess {
-                messageRepository.getMessageById(conversationId, messageId)
-                    .onSuccess { message ->
-                        val assetToRemove = when (message.content) {
-                            is MessageContent.Asset -> (message.content as MessageContent.Asset).value.remoteData
-                            else -> null
-                        }
-                        if (assetToRemove != null) {
-                            assetRepository.deleteAsset(
-                                AssetId(assetToRemove.assetId, assetToRemove.assetDomain.orEmpty()),
-                                assetToRemove.assetToken
-                            )
-                                .onFailure {
-                                    kaliumLogger.withFeatureId(ASSETS).w("delete message asset failure: $it")
-                                }
-                        }
-                    }
-            }
+            .onSuccess { removeMessageAsset(conversationId, messageId) }
```

Comment:
Could you update the unit test in `DeleteMessageUseCaseTest` based on your changes ?

Reply:
I think I don't need to do that because removing locally logic is inside asset repository so in use case it doesn't change anything